src/private/WRLDRoutingServiceHelpers.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "RoutingQueryResponse.h"
+#include "WRLDRoutingQueryResponse.h"
+
+namespace Eegeo
```

Comment:
Not sure what this is - it looks like it's mimicking the C++ platform namespaces but is actually an objective-c factory class method.  Given this is in the obj-c api, we should probably define this as an obj-c class method so it's defined like:
`+ (WRLDRoutingQueryResponse*)createWRLDRoutingQueryResponse :(const `Eegeo::Routes::Webservice::RoutingQueryResponse*)withResponse;
and invoked like:
`[WRLDRoutingServiceHelpers createWRLDRoutingQueryResponse:result]`

if that's possible